.github/workflows/windows.yml

```diff
@@ -25,29 +25,24 @@ jobs:
 
     env:
       BITS: '${{ matrix.bits }}'
-      ICINGA_BUILD_TYPE: snapshot
-      UPSTREAM_GIT_URL: file://D:/a/icinga2/icinga2/.git
+      CMAKE_BUILD_TYPE: RelWithDebInfo
 
     steps:
       - name: Checkout HEAD
         uses: actions/checkout@v1
 
-      - name: windows-icinga2
-        run: |
-          git clone https://git.icinga.com/packaging/windows-icinga2.git
-
       - name: Build tools
         run: |
+          Set-PSDebug -Trace 1
           & .\doc\win-dev.ps1
 
-      - name: Source
-        run: |
-          git checkout -B master
-          cd windows-icinga2
-          & .\source.ps1
-
       - name: Binary
-        working-directory: windows-icinga2
         run: |
-          New-Item -ItemType Directory -Path 'C:\Program Files\Icinga2\WillBeRemoved' -ErrorAction SilentlyContinue
```

Comment:
Why is that temporary directory no longer needed? Additionally, why are the clean-ups performed by the previous script from GitLab no longer required?
```bash
Remove-Item Build\Icinga*.msi
Remove-Item -Recurse "C:\Program Files\Icinga2\*"
```

Reply:
> Why is that temporary directory no longer needed?

The no longer used build.ps1 from GitLab did `Remove-Item -Recurse "C:\Program Files\Icinga2\*"` which I've omitted.

> Additionally, why are the clean-ups performed by the previous script from GitLab no longer required?
> 
> ```shell
> Remove-Item Build\Icinga*.msi
> Remove-Item -Recurse "C:\Program Files\Icinga2\*"
> ```

This is a GHA. It cleans up itself.

Reply:
> This is a GHA. It cleans up itself.

No! 🙈 This is a cleanup of previous builds.

But the argument is the same:

The previous build is also a GHA. It cleans up itself.